packages/geojson-extension/package.json

```
@@ -29,7 +29,10 @@
     "lib/*.d.ts",
     "lib/*.js",
     "style/*.*",
```

That should be enough to include everything in the `style` folder:

```suggestion
    "style/**/*",
```